src/ensembl/src/content/app/species/components/species-main-view/SpeciesMainView.scss

```diff
@@ -6,5 +8,124 @@
 }
 
 .speciesLabelBlock {
-  padding: 0 75px 0 60px;
+  padding: 0 75px 0 45px;
+}
+
+// TODO: Remove Me
+.statsWrapper { 
```

Comment:
TODO: This needs to be updated once the PR https://github.com/Ensembl/ensembl-client/pull/351 is merged